src/plot.js

```diff
@@ -171,11 +171,19 @@ export function plot(options = {}) {
     return {...state, channels: {...state.channels, ...facetState?.channels}};
   };
 
+  class StickyEvent extends Event {
+    constructor(type, options) {
+      super(type, options);
+      this.sticky = options.sticky;
+    }
+  }
```

Comment:
This should be moved outside the plot function so there’s a single (global) class.